util/odemis-park-mirror

```diff
@@ -135,4 +145,5 @@ def main(args):
 
 if __name__ == '__main__':
     ret = main(sys.argv)
+    time.sleep(5)
```

Comment:
Whats the reason for the sleep?

Reply:
Good catch. It was for debug. I forgot it... fixed now!